api/http/workflow.go

```diff
@@ -214,12 +226,78 @@ func (h WorkflowController) StartWorkflow(c *gin.Context) {
 	}
 
 	c.JSON(http.StatusCreated, gin.H{
-		"id":   job.ID,
+		"id":   jobID,
 		"name": name,
 	})
 	l.Infof("StartWorkflow starting workflow %s : StatusCreated", name)
 }
 
+// StartBatchWorkflows responds to POST /api/workflow/:name/batch
+func (h WorkflowController) StartBatchWorkflows(c *gin.Context) {
+	l := log.FromContext(c.Request.Context())
+
+	var name string = c.Param("name")
+	l.Infof("StartWorkflow starting workflow %s", name)
+
+	var inputParametersBatch []map[string]interface{}
+	if err := c.BindJSON(&inputParametersBatch); err != nil {
+		c.JSON(http.StatusBadRequest, gin.H{
+			"error": fmt.Sprintf("Unable to parse the input parameters: %s",
+				err.Error()),
+		})
+		return
+	}
+
+	result := make([]map[string]string, 0, len(inputParametersBatch))
+	jobs := make([]struct {
+		jobJSON []byte
+		jobID   string
+		subject string
+	}, 0, len(inputParametersBatch))
+	for _, inputParameters := range inputParametersBatch {
+		jobJSON, jobID, subject, err := h.startWorkflowGetJob(c, inputParameters, name)
+		if err != nil {
+			l.Error(err)
+			statusCode := http.StatusBadRequest
+			switch err {
+			case store.ErrWorkflowNotFound:
+				statusCode = http.StatusNotFound
+			default:
+			}
+			c.JSON(statusCode, gin.H{
+				"error": err.Error(),
+			})
+			return
+		}
+		jobs = append(jobs, struct {
+			jobJSON []byte
+			jobID   string
+			subject string
+		}{
+			jobJSON: jobJSON,
+			jobID:   jobID,
+			subject: subject,
+		})
+	}
+
+	for _, job := range jobs {
+		jobResult := map[string]string{
+			"id":   job.jobID,
+			"name": name,
+		}
+		err := h.nats.JetStreamPublish(job.subject, job.jobJSON)
+		if err != nil {
+			l.Error(errors.Wrap(err, "JetStreamPublish failed"))
+			jobResult["id"] = ""
```

Comment:
@alfrunes I don't get your suggestion...

Reply:
I thought we'd return a simple error JSON object on error:
```json
{"error": "message"}
```
instead of
```json
{"id": "", "name": "<name>", "error": "message"}
```
But either one is not violating the API specs, so I'm not sure what the intention is...

Reply:
ok, got it!